apps/experiments/tests/test_versioning.py

```
@@ -75,6 +75,59 @@ def test_compare(self):
         assert changed_field.changed is True
         assert changed_field.previous_field_version.raw_value == 0.2
 
+    @pytest.mark.django_db()
```

Ruff

Use `@pytest.mark.django_db` over `@pytest.mark.django_db()`

Remove parentheses

(PT023)